### PR TITLE
Feature Gates: Add steps to verify

### PR DESCRIPTION
**Pull request by mburke5678**
@rioliu-rh Suggested how QE verifies feature sets in https://github.com/openshift/openshift-docs/pull/42124#issuecomment-1049417624. Adding steps to the docs here.

### CODE REVIEW

**Comment by mburke5678** [comment]
@rioliu-rh can you please take a look? Thank you in advance.

File: modules/nodes-cluster-enabling-features-cli.adoc [hunk 1]
```diff
@@ -40,7 +40,53 @@ spec:
 * `TechPreviewNoUpgrade` enables specific Technology Preview features.
 --
 +
+After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled as the change is being applied.
```
Comment (michaelryanpeter) [inline modules/nodes-cluster-enabling-features-cli.adoc:43]:
```suggestion
After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
```
Re: [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a) entry on _as_

File: modules/nodes-cluster-enabling-features-console.adoc [hunk 2]
```diff
@@ -41,7 +41,51 @@ spec:
 * `TechPreviewNoUpgrade` enables specific Technology Preview features.
 --
 +
+After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled as the change is being applied.
```
Comment (michaelryanpeter) [inline modules/nodes-cluster-enabling-features-console.adoc:44]:
```suggestion
After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
```
Re: [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a) entry on _as_

**Review by michaelryanpeter — COMMENTED** [review]
The same nit from the IBM Style guide repeated. Otherwise, LGTM.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.11` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.10. And any PR going into `main` must also target the latest version branch (enterprise-4.11).

If the update in your PR does NOT apply to version 4.11 onward, please retarget this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request merged**